docs/algorithms/sig/sphincs.yml

```diff
@@ -26,7 +26,7 @@ nist-round: 3
 spec-version: NIST Round 3 submission, v3.1 (June 10, 2022)
 spdx-license-identifier: CC0-1.0
 primary-upstream:
-  source: https://github.com/PQClean/PQClean/commit/d742438e5c541958bfd58070cd8668d757d88e07
+  source: https://github.com/PQClean/PQClean/commit/66e50172055aaf1b9a16d8f35fe03b0807f2723e
```

Comment:
Similar (license documentation) question here: This states CC0 -- which is textually correct looking at the immediate [LICENSE upstream in PQClean](https://github.com/PQClean/PQClean/blob/master/crypto_sign/sphincs-sha2-256s-robust/clean/LICENSE). When looking at the ultimate origin code base [Sphincs+ is multi licensed](https://github.com/sphincs/sphincsplus/blob/master/LICENSE). If we manually "curate" license statements (see comment on kyber above) why not do the same with Sphincs+? This would resolve #1437 without having to push LICENSE statements "through" PQClean.

Reply:
Not sure if this is possible if PQClean only lists CC0.

Reply:
In that case [manual curation](https://github.com/open-quantum-safe/liboqs/pull/1512#discussion_r1271902542) in general should stop and we ought to move to full automation (see #1514).

Reply:
Agreed. However, there seems to be a nuance when curating the information (no matter if manual or automatic):
(1) Kyber/aarch64 case: PQClean LICENSE file lists one license and the PQClean source files contain more/other licenses.
(2) SPHNCS+ case: PQClean code base lists a single license. The code's ancestor contains more/other licenses.

Reply:
"Manual" curation allows (arguably/hopefully legally informed) updates, automated license collection does not. In either case, work is required (whether already in this PR or only when/if resolving #1514, I leave to the reviewers of this PR).